### PR TITLE
update nature settings for global-meta

### PR DESCRIPTION
**Pull request by ricky-lau**
Update Nature settings to match existing c-meta style from app (and support it being removed from nature app)

### CODE REVIEW

File: toolkits/global/packages/global-meta/scss/10-settings/_nature.scss [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 $meta--font-family: $font-family-sans;
-$meta--border: 1px solid greyscale(80);
-$meta--color: greyscale(40);
-$meta--margin-bottom: 4px;
+$meta--border: 1px solid #bbb;
+$meta--color: inherit;
+$meta--line-height: 1.4;
+$meta--margin-bottom: spacing(4);
+$meta--margin-right: spacing(0);
+$meta--padding-right: spacing(4);
```
Comment (tulipdexter) [inline toolkits/global/packages/global-meta/scss/10-settings/_nature.scss:16]: So Nature meta will _always_ want this padding to the right?

Comment (ricky-lau) [reply toolkits/global/packages/global-meta/scss/10-settings/_nature.scss:16]: Yeah and the component wont apply this to the last child.

Unless you mean we have a situation ive missed where the meta is right aligned?

Comment (tulipdexter) [reply toolkits/global/packages/global-meta/scss/10-settings/_nature.scss:16]: This is spacing between meta things? Why padding and not margin?

Comment (ricky-lau) [reply toolkits/global/packages/global-meta/scss/10-settings/_nature.scss:16]: Oh I see. It's for the spacing of border that separates between items

**Review by tulipdexter — APPROVED** [review]

**Pull request merged**